### DISCUSSION TITLE
g4 file for the _visitor.py file

### DISCUSSION THREAD

**Discussion by maxliu** [fugue-project/fugue#347] [Q&A]
Hi 

I was wondering where I could find the g4 file for this fugue_sql/_visitors.py file.
Thanks.

**Comment by kvnkho** [comment]
Hi @maxliu, it was refactored out. Check [here](https://github.com/fugue-project/fugue-sql-antlr/blob/master/fugue_sql.g4). Is there something specific you are looking for?
